rivers/yahoo-finance-prices/parser.js

```diff
@@ -0,0 +1,26 @@
+var _ = require('lodash'),
+    moment = require('moment-timezone');
+
+module.exports = function(config, body, url, temporalDataCallback, metaDataCallback) {
+    var data = JSON.parse(body);
+
+    // This is important.
+    moment.tz.setDefault("UTC");
```

Comment:
No, use the timezone set in your config. 

    moment.tz.setDefault(config.timezone);